src/module/interface/update/index.ts

```diff
@@ -1,11 +1,16 @@
 import { Node } from 'slate';
 
 export interface IUpdate {
+  text: Node[];
   updateId: string;
   userId: string;
   timelineId: string;
-  text: Node[];
-  numberValue: number;
-  isFlipped: boolean;
-  isContext: boolean;
 }
+
+export interface INewUpdate {
+  text: string;
+  audienceId: string,
+  organizationId: string,
+  timelineId: string;
+  userId: string;
+}
```

Comment:
Please make sure to make your editor keep the newlines at the end of files. That is like an unwritten law. The next developer will revert the change automatically. We can spare the back and forth. ❤️

Reply:
I just started using the VS Code editor - will make adjustment.